test/testunit-test-util.rb

```diff
@@ -24,7 +24,11 @@ def _run_test(test_case, name)
     result = Test::Unit::TestResult.new
     test = test_case.new(name)
     yield(test) if block_given?
-    test.run(result) {}
+    suite = test_case.suite
```

Comment:
It seems that this may include needless tests.
Can we create an empty test suite instead?

```suggestion
    suite = Test::Unit::TestSuite.new(test_case.name, test_case)
```

Reply:
Certainly, this may append needless tests:

https://github.com/test-unit/test-unit/blob/73d5a703d346fadd712d1139c2f0945a23826075/lib/test/unit/test-suite-creator.rb#L24-L40

Thanks!